defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+summon:
+  version: 0.6.6
+  os: linux-amd64
+summon_conjur:
+  version: 0.5.0
+  os: linux-amd64
```

Comment:
can we at least bump these to the latest versions available, if we're going to hardcode these values for now?

Reply:
Absolutely